### DISCUSSION TITLE
Running apache ozone from Intellij

### DISCUSSION THREAD

**Discussion by kiruthiga1793** [apache/ozone#7629] [FAQ]
Those who run ozone from IntelliJ(windows) , is there any pre requisites for that? I am getting this error while building.

![image](https://github.com/user-attachments/assets/00b92870-0c55-4d94-8118-6611c2dbffad)

Suggest me some ways to fix it.

**Comment by nandakumar131** [comment]
Are you able to build the project via command line?
Also check if `hadoop-ozone/dev-support/checks/docs.sh` script has execute permission.

**Reply by kiruthiga1793** [reply]
@nandakumar131 I tried to build the project from command line. 
![image](https://github.com/user-attachments/assets/3566aca0-470c-452e-8bcc-1e7df79d861b)

It says the hadoop-ozone/dev-support/checks/docs.sh is not found. But, it is actually there in my system. Suggest me some ways to fix this please. I am newbie to this project.

**Reply by nandakumar131** [reply]
This looks more like the environmental issue rather than problem in the build script.

There is a space in the path where you've cloned the repository, can you try moving the codebase to different location or rename the `kiru ozone` directory to something like `kiru_ozone`?

Tip: Please avoid using screenshots for sharing code/output. It becomes hard to select or copy part of the shared code/output.
You can use `code block` for this. 
Ref: [Quoting code](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)

**Reply by kiruthiga1793** [reply]
@nandakumar131 I changed the directory to kiru_ozone and ran the build again. The same error still exists. Will I be needing any other environmental setup to run in Windows?

**Reply by nandakumar131** [reply]
Can you check if you have `hadoop-ozone/dev-support/checks/docs.sh` script present and has execute permission set?
